src/tool/subcommands/archive_cmd.rs

```diff
@@ -138,6 +145,18 @@ pub enum ArchiveCommands {
         #[arg(long, default_value_t = false)]
         force: bool,
     },
+    /// Merge a v1 Filecoin snapshot with an F3 snapshot into a v2 Filecoin snapshot in `.forest.car.zst` format
+    MergeF3 {
+        /// Path to the Filecoin snapshot
+        #[arg(long)]
+        filecoin: PathBuf,
```

Comment:
Can a `v2` snapshot also be provided here? If not, the `v1` name should probably be used instead.

Reply:
Fixed.